lsp-completion.el

```diff
@@ -740,7 +740,7 @@ The CLEANUP-FN will be called to cleanup."
              (fboundp 'company-mode))
         (setq-local company-abort-on-unique-match nil)
         (company-mode 1)
-        (add-to-list 'company-backends 'company-capf))
+        (setq-local company-backends (cl-adjoin 'company-capf company-backends :test #'cl-equalp)))
```

Comment:
The `cl-equalp` is comparing string case-insensitively, I think you should just use `equal` here.
Or even better, since we're comparing symbol, the `:test` stuff might be omitted?

Reply:
I agree `equal` should be used. I had just used `cl-equalp` first. Omitting `:test` uses `eql` and that does not compare symbols. I'll update now.

Reply:
actually, in this case it would not make a difference. `cl-equalp` will use `equal` when comparing symbols.